pxr/base/plug/CMakeLists.txt

```diff
@@ -166,6 +166,21 @@ pxr_create_test_module(TestPlugModuleIncomplete
     INSTALL_PREFIX PlugPlugins
 )
 
+if (CMAKE_SYSTEM_NAME STREQUAL "Windows")
+    # Create a test module that has a unicode character in it's path
+    # for Windows
+    pxr_create_test_module(TestPlugModuleUnicode
+        INSTALL_PREFIX PlugPlugins
+        DEST_DIR TestPlugMödüleÜnicöde
```

Comment:
It's worth pointing out that `DEST_DIR` will be a created directory which contains unicode characters to verify the fix